.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint/eslint-plugin'],
+  extends: [
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier',
+    'prettier/@typescript-eslint',
+    'plugin:prettier/recommended',
+    ''
+  ],
+  root: true,
+  env: {
+    node: true,
+  },
+  rules: {
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-unused-vars': ["warn", { "argsIgnorePattern": "^_" }],
+    'no-return-await': 'warn',
+    'no-throw-literal': 'warn'
+  },
+};
```

Comment:
Is this file still required? It looks the same as the one in the `test/` directory.

Reply:
Not needed, removed